packages/compiler/src/core.ts

```diff
@@ -550,3 +550,7 @@ export const enum AttributeMarker {
    */
   I18n = 6,
 }
+
+// Pasted from render3/interfaces/view.ts
+/** Index at which the context of an LView is stored. */
```

Comment:
Just a small proposal on docs:
```suggestion
/**
 * Index at which the context of an LView is stored.
 * Note: it **must** be in sync with corresponding const in packages/core/src/render3/interfaces/view.ts.
 */
```